app/pages/app/app.js

```diff
@@ -433,6 +436,14 @@ export function mapStateToProps(state) {
       );
     }
 
+  if (state.blip.membershipPermissionsInOtherCareTeams ) {
```

Comment:
Recommend that we place this logic within the above `if (state.blip.currentPatientInViewId) { ... }` block, as we only want to try to set it if we have `state.blip.currentPatientInViewId` available.